### ISSUE TITLE
Proposed: Publish procedures as markdown

### ISSUE THREAD

**Issue by briesenberg07** [UniverityOfWashingtonLibrariesSchemas/schemasProject#20]
Greetings @gerontakos 

I'd like to continue building out the documentation that you've started for schemasProject. I should not spend too much time on this, but I am currently asking and getting answers to a lot of questions about how to use these tools--I may as well add to the documentation as I'm able and while the process is fresh in my mind. I believe that we would do well to have a more complete set of how-to documentation here, even if a revamp is in mind (we don't know when).

I propose using a [GitHub wiki](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/documenting-your-project-with-wikis) to do this. This would mean moving files in `procedures/` to a wiki repository and managing separately (GitHub wikis are tracked separately from the repository they are associated with; they are their own repository). This would allow us to write in [markdown](https://guides.github.com/features/mastering-markdown/), not HTML. Writing in markdown is preferable to me for creating documentation. 

It did occur to me that the documentation was in HTML because you were, or were planning to, generate it using a transform or the like?

**Comment by gerontakos** [comment]
At this point, I don't think it matters too much how we do the documentation (html v markdown). The original vision was that all work would be done using web browsers and xml editors. github wiki docs can be viewed in a browser too of course. But I can work with a github wiki, if you prefer that.

**Comment by briesenberg07** [comment]
Thank you for weighing in @gerontakos !
Since you are amenable I may forge on with the GitHub wiki plan. I'm happy to handle converting existing content to markdown files (a.k.a. copy/paste, and add some \# headings and \- bullets, organize a bit, etc.).

**Comment by briesenberg07** [comment]
@gerontakos 
I converted draft procedure files to markdown. Content is unchanged. I reproduced most HTML formatting (and added some additional formatting) in markdown. Some additional details:

- A [README](https://github.com/UniverityOfWashingtonLibrariesSchemas/schemasProject/blob/master/README.md) was created and includes a tagline and links to documentation, which can be viewed in a browser without opening local copies of HTML files.
- I placed markdown files in [`procedures/`](https://github.com/UniverityOfWashingtonLibrariesSchemas/schemasProject/tree/master/procedures), not in a wiki, because I like using HackMD to edit and push changes and this isn't possible with GitHub wikis
- Documentation can be edited by pulling/pushing from/to `master`, or using HackMD (I can provide info on this if it is something you are interested in).
